.github/.OwlBot.yaml

```diff
@@ -1027,6 +1029,8 @@ deep-copy-regex:
     dest: /
   - source: /google/cloud/speech/v2/cloud.google.com/go
     dest: /
+  - source: /google/storage/v2/cloud.google.com/go/storage/control/apiv2
+    dest: /
   - source: /google/storage/v2/cloud.google.com/go/storage/internal/apiv2
     dest: /storage/internal/apiv2
```

Comment:
I'm not sure why the internal GAPIC for the data plane API has a non-`/` value for the `dest` ...any ideas?

Reply:
Not sure if this question is for Storage team; but could you clarify what you mean by `non-/` value?

Reply:
This was a question for @quartzmo (he's more experienced with this config than me, I'm trying to reacquaint myself).

If you look at most other entries, the `dest` is just `/`. This is the "deep copy" destination. And in talking this through with you in this comment I realize I need to set `dest` to `/storage/control/apiv2` :)

Thanks for rubber 🦆 'ing